admin/installer/cloudformation.py

```diff
@@ -150,25 +150,32 @@ def _validate_cluster_size(size):
 ))
 secret_access_key_param = template.add_parameter(Parameter(
     "AmazonSecretAccessKey",
-    Description="Your Amazon AWS secret access key (mandatory)",
+    Description="Required: Your Amazon AWS secret access key",
     Type="String",
     NoEcho=True,
     MinLength="1",
 ))
 keyname_param = template.add_parameter(Parameter(
     "EC2KeyPair",
-    Description="Name of an existing EC2 KeyPair to enable SSH "
-                "access to the instance (mandatory)",
+    Description="Required: Name of an existing EC2 KeyPair to enable SSH "
+                "access to the instance",
+    Type="AWS::EC2::KeyPair::KeyName",
+))
+secret_access_key_param = template.add_parameter(Parameter(
```

Comment:
Sorry, I missed this.
We're overwriting the `secret_access_key_param` variable.
Which means that the node cloud-init scripts get `secret_access_key="Yes"`.

We need to fix this and redeploy the cloudformation template.